ruff.schema.json

```diff
@@ -1662,6 +1695,17 @@
             "$ref": "#/definitions/RuleSelector"
           }
         },
+        "extend-unfixable": {
```

Comment:
We shouldn't remove deprecated options from the schema to avoid existing configurations failing the schema validation.